docs/content/en/docs/Development/nvidia.md

```diff
@@ -0,0 +1,269 @@
+---
+title: "Booting Kairos on Nvidia Jetson ARM"
+linkTitle: "Booting Kairos on Nvidia Jetson ARM"
+weight: 5
+date: 2022-11-13
+description: >
+    This page contains a reference on how to run Kairos on Nvidia Jetson ARM
+---
+
+{{% alert title="Note" %}}
+Please note that the following page contains only development reference. At the time of writing, we have tried porting Kairos to Jetson Nano eMMC without success. This is due to the old kernel supported (4.9), not properly working with `EFISTUB` and `U-boot` (you can see the [issue here](https://github.com/kairos-io/kairos/issues/45)). However, the steps outlined _should_ be a good reference to port Kairos to those architecture _when_ a new kernel version is available. We have tested, and have successfully booted a Jetson Nano with the 5.15 kernel, however, due to the lack of driver support, eMMC partitions are not properly recognized.
+{{% /alert %}}
+
+This page is a development reference in order to boot Kairos in Nvidia Jetson devices. Nvidia Jetson images by default ship `extlinux` as bootloader, without EFI boot. This guide explains how to get instead u-boot to chainload to `grub2`, which can be used to boot and load `Kairos`.
+
+Note that currently there are no official Kairos core images for Jetson images, this page will refer to Jetson Nano eMMC version as the current reference, but the steps should be similar, as outline how to use the Nvidia SDK to flash the OS onboard in the eMMC of the device.
```

Comment:
```suggestion
Note that currently there are no official Kairos core images for Jetson images. This page will refer to Jetson Nano eMMC version as the current reference, but the steps should be similar, as outline how to use the Nvidia SDK to flash the OS onboard in the eMMC of the device.
```

Reply:
Which steps are "similar" to what? This sentence implies there is some other version but it's not implicitly mentioned.